plugin/core/views.py

```diff
@@ -885,40 +885,45 @@ def format_completion(
     lsp_label = item["label"]
     lsp_label_details = item.get("labelDetails")
     lsp_filter_text = item.get("filterText")
-    st_annotation = (item.get("detail") or "").replace('\n', ' ')
-
-    st_details = ""
+    st_annotation = ''
+    details = []  # type: List[str]
     if can_resolve_completion_items or item.get("documentation"):
-        st_details += make_command_link("lsp_resolve_docs", "More", {"index": index, "session_name": session_name})
+        details.append(make_command_link("lsp_resolve_docs", "More", {"index": index, "session_name": session_name}))
     if lsp_label_details:
-        if st_details:
-            st_details += " | "
         lsp_label_detail = lsp_label_details.get("detail")
         lsp_label_description = lsp_label_details.get("description")
-        st_details += "<p>"
-        # `label` should be rendered most prominent.
-        st_details += _wrap_in_tags("b", lsp_label)
+        label_detail = lsp_label
         if isinstance(lsp_label_detail, str):
             # `detail` should be rendered less prominent than `label`.
             # The string is appended directly after `label`, with no additional white space applied.
-            st_details += html.escape(lsp_label_detail)
+            label_detail += lsp_label_detail
+        # annotations.append(label_detail)
+        lsp_label = label_detail
```

Comment:
Can we avoid having this?
```
label_detail = lsp_label
...
lsp_label = label_detail
```

Reply:
It got a bit messy after many back-and-forth. Cleaned up now.